examples/BasicUsage_clr21_RBAC/k8s/saRole.yaml

```diff
@@ -0,0 +1,24 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  # "namespace" omitted since ClusterRoles are not namespaced
+  name: metrics-reader
+rules:
+- apiGroups: ["","extensions"]
```

Comment:
Fair guess. In this case, "" really means "". :-) 
Refer the doc: [Using RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/):
**# "" indicates the core API group**

The binding above leads to the following cluster role. "deployments.extensions" / "replicasets.extensions", etc, is there because **extensions** is explicitly set:

`kubectl describe clusterrole metrics-reader`
```
Name:         metrics-reader
Labels:       <none>
Annotations:  <none>
PolicyRule:
  Resources               Non-Resource URLs  Resource Names  Verbs
  ---------               -----------------  --------------  -----
  deployments             []                 []              [get watch list]
  deployments.extensions  []                 []              [get watch list]
  nodes                   []                 []              [get watch list]
  nodes.extensions        []                 []              [get watch list]
  pods                    []                 []              [get watch list]
  pods.extensions         []                 []              [get watch list]
  replicasets             []                 []              [get watch list]
  replicasets.extensions  []                 []              [get watch list]
```

Reply:
Got it, thanks!